api/src/accounts.rs

```diff
@@ -664,7 +664,8 @@ impl Account {
             .find_resource(&state_view, self.address, resource_type)
             .context(format!(
                 "Failed to query DB to check for {} at {}",
-                resource_type, self.address
+                resource_type.to_canonical_string(),
```

Comment:
Wondering: why didn't we implement Display via to_canonical_string and make them equivalent? Want to have this explicit?